Cakes-Co/style.css

```diff
@@ -1 +1,257 @@
 /*  Add your styling here */
+
+*{
+    margin: 0;
+    box-sizing: border-box;
+    padding: 0;
+}
+
+body{
+
+    
+    font-family: "Poppins";
+}
+
+.header{
+background-image: url("https://images.pexels.com/photos/7966383/pexels-photo-7966383.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1" );
+background-position: center;
+background-size: cover;
+background-repeat: no-repeat;
+padding: 150px;
+position:relative;
+z-index: 2;
+text-align: center;
+}
+
+h1{
+    color: rgba(218, 213, 213, 0.979);
+    z-index: 3;
+    position: relative;
+     bottom: 50px;
+  
+}
+
+.gray-cover{
+    background-color: rgba(5, 2, 2, 0.438);
+    position: absolute;
+    z-index: 2;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+}
+
+.navbar{
+    margin: 20px;
+    padding: 20px;
+    background-color:rgba(250, 235, 215, 0.671);
+    border-radius: 2px;
+   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+ 
+}
+
+.navlist, a{
+    list-style-type: none;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    text-decoration: none;
+    color: black ;
+    font-weight: bold;
+    
+}
+
+a:hover{
+    color: rgb(104, 54, 54);
+}
+
+
+
+
+.main-container{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    line-height: 1.8em;
+    justify-content: space-evenly;
+    background-color: rgba(250, 235, 215, 0.671);
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    margin: 1em;
+    padding:1em ;
+
+}
+
+.main-text{
+    color:rgb(104, 54, 54);
+    
+    
+}
+
+
+
+
+.aside-img{
+    width:300px;
+    border-radius:3px;
+   
+    
+}
+
+.text-section{
+    
+    display: flex;
+
+    flex-direction: column;
+    align-items: center;
+    padding: 1em;
+
+    width: 330px;
+
+}
+
+
+
+
+  .cake-section {
+        display: flex;
+         text-align: center;
+    gap: 10px;
+    padding: 10px;
+    margin: 9px;
+    justify-content:space-evenly;
+
+
+    }
+
+
+    .cake-item {
+        position: relative;
+        overflow: hidden;
+    }
+
+    .select-cake {
+  width: 200px;
+    border-radius: 3px;
+    
+    }
+
+    .overlay {
+        position: absolute;
+        bottom: 7px;
+        background-color: rgba(0, 0, 0, 0.6);
+        color: white;
+        width: 100%;
+        text-align: center;
+        padding: 10px;
+        visibility: hidden;
+        opacity: 0;
+        transition: opacity 0.3s ease, visibility 0.3s ease;
+        cursor: pointer;
+    }
+
+    .cake-item:hover .overlay {
+        visibility: visible;
+        opacity: 1;
+    }
+
+    .text-container h1 {
+            color: rgb(104, 54, 54);
+             height: 2px;
```

Comment:
Is there any reason to set `height: 2px;` here? It looks a bit tiny for the `h1` to fit in.
<img width="715" alt="Screenshot 2024-10-20 at 15 14 12" src="https://github.com/user-attachments/assets/280abe95-aa7c-4429-beff-876e49ccc0c1">